js/core.js

```diff
@@ -2046,9 +2044,16 @@
                     columnOrder = annotation.column_order;
                 }
 
+                if (typeof annotation.pre_format === 'object') {
+                    if (typeof annotation.pre_format.format !== 'string') {
```

Comment:
You don't need to throw an error, just ignore the annotation as if it didn't exist. You can just do a `console.log`.

Reply:
Alright I have removed it.